src/ggcomm.c

```diff
@@ -1344,6 +1346,26 @@ concatenate_results (RESULT_NODE *r,
 
     rc [totlines * totlinelength] = 0;
 
+    /*  Trim whitespace from all lines  */
+    txt = rc;
+    while (txt)
+      {
+         line_end = strchr(txt, '\n');
+         if (!line_end)
+           break;
+         if (*(line_end - 1) == '\r')
```

Comment:
I tested by converting a CZMQ .api file to CRLF via unix2dos, and regenerating. It seemed to do its job, but I don't have a Windows dev box so that's all I can do.

If problems are reported it can be backed out again. Thanks!